src/Microsoft.TestPlatform.CommunicationUtilities/Interfaces/ITestRequestHandler2.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.CommunicationUtilities.Interfaces
+{
+    public interface ITestRequestHandler2 : ITestRequestHandler
```

Comment:
That is true. I dont see why interfaces in assemblies other than the Translation Layer and the ObjectModel need to be public at all. Just in case I'm missing a scenario a github search for this interface, doesn't reveal any intent of this needing to be public.

Reply:
Can we make the new one internal nevertheless?

Reply:
Only concern would be if the old one is public for a reason - but I agree it may be best to at least make the new ones internal... I'd rather that if we can confirm these interfaces are truly internal, we can mark them both internal.

Reply:
Removed ITestRequestHandler2 interface and mutated ITestRequestHandler.